models/airports.malloy

```diff
@@ -17,31 +17,31 @@ source: airports is duckdb.table('../src/documentation/data/airports.parquet') e
     pick 'Western Pacific' when 'AWP'
     pick 'Alaska' when 'AAL'
 
-  query: by_region is -> {
```

Comment:
Looks like you removed the arrow in all these places without also removing one space. Can you maybe do a find and replace for `is  {` (two spaces) -> `is {` (one space)?